### DISCUSSION TITLE
Octo Ignores default repo :(

### DISCUSSION THREAD

**Discussion by Janik-Haag** [pwntester/octo.nvim#420] [General]
Hey I have a repo setup with multiple remotes some of them on github. I use `gh repo set-default` to set the default repo and then I can for example use `gh pr checkout #pr` and it works greate but octo uses the the git origin which is not what I want because I want to open PRs from my fork and not push to the main repo directly by accident. Does anyone have a solution for this?

**Comment by pwntester** [comment]
You can specify the order in which Octo tries different remotes with `default_remote`. The default value is `{"upstream", "origin"}`. You cant change that at the repo level though. Its a global setting

**Reply by Janik-Haag** [reply]
Ah thank you, sorry for the noise
